cmd/spread/main.go

```diff
@@ -34,6 +34,8 @@ var (
 	seed           = flag.Int64("seed", 0, "Seed for job order permutation")
 	repeat         = flag.Int("repeat", 0, "Number of times to repeat each task")
 	garbageCollect = flag.Bool("gc", false, "Garbage collect backend resources when possible")
+	xunit          = flag.Bool("xunit", false, "Create a XUnit report")
```

Comment:
Just a question, does it make sense to have both, or could we just go with xunit?

Reply:
well, this has to be discussed.
I created both because the idea of xunit is to use it to display in the actions page, and the jsonunit is to query

Reply:
Perhaps we could have just jsonunit and then convert to xunit format with a tool, so we don't need the xunit report.
@bboozzoo does it make sense?